### PR TITLE
feat: About activity compose dialogs and extensions

### PR DESCRIPTION
**Pull request by FunkyMuse**
- add extensions for showing app launched dialogs
- migrate About activity to use Compose dialogs

### CODE REVIEW

File: commons/src/main/kotlin/com/simplemobiletools/commons/compose/extensions/AcitivtyExtensions.kt [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    }
+
+    baseConfig.lastVersion = currVersion
+}
+
+fun ComponentActivity.upgradeToPro() {
+    launchViewIntent("https://simplemobiletools.com/upgrade_to_pro")
+}
+
+const val DEV_PS_URL = "https://play.google.com/store/apps/dev?id=9070296388022589266"
```
Comment (tibbi) [inline commons/src/main/kotlin/com/simplemobiletools/commons/compose/extensions/AcitivtyExtensions.kt:89]: whats PS? This isnt  a clear constant at all... Make it DEVELOPER_URL or so

**Comment by tibbi** [comment]
when the use clicks on "Rate us" on About page, then Skip, we should show the RateStars dialog, not redirect directly.

**Comment by FunkyMuse** [comment]
> when the use clicks on "Rate us" on About page, then Skip, we should show the RateStars dialog, not redirect directly. Redirect the user only if he picks 5 stars.

Done

**Comment by tibbi** [comment]
ok thanks, should be good. Will do more testing in a real app.

**Comment by FunkyMuse** [comment]
Will merge this into thank you

**Comment by sudoshindo** [comment]
@tibbi sellout

**Pull request merged**